frontend/src/utils/files.ts

```diff
@@ -0,0 +1,52 @@
+import { File, FileTree } from '../types'
+
+export const parseToFileTree = (files: File[], rootOffset: number = 2) => {
```

Comment:
Should we start frontend testing with testing these utils functions?

Reply:
I think that's a good idea. I can try to write some tests.